datahub-ranger-plugin/README.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Configuring Apache Ranger"
+hide_title: true
+---
+# Configuring Apache Ranger
+Datahub Apache Ranger Plugin configuration is consist of below configuration sections 
+1. Configuration at Apache Ranger Deployment 
+2. Configuration at Datahub Deployment
+
+The configuration sections are tested on [Privacera Platform](https://privacera.com/) v6.3.0.1.
+
+# Assumption 
+- Apache Ranger and DataHub are configured with same IDP
+- Apache Ranger service is running on HTTP
+
+# Configuration 
+## Configuration at Apache Ranger Deployment 
+1. Download the datahub-ranger-plugin from [Maven](https://mvnrepository.com/artifact/io.acryl/datahub-ranger-plugin)
+2. Create  *datahub*  directory in each *Privacera Ranger* container 
+```bash
+docker exec privacera_ranger_1 mkdir ews/webapp/WEB-INF/classes/ranger-plugins/datahub
+```
+3. Copy the datahub-ranger-plugin jar into *Privacera Ranger* container
+```bash 
+docker cp datahub-ranger-plugin-<version>.jar privacera_ranger_1:/opt/ranger/ranger-2.1.0-admin/ews/webapp/WEB-INF/classes/ranger-plugins/datahub/
+```
+4. Download the [service definition](../datahub-ranger-plugin/conf/servicedef.json). This service definition is the ranger service definition JSON file for datahub-ranger-plugin-<version\>.jar
+5. Execute below curl command to register the service definition <br /> 
+Replace variables with corresponding values in curl command
+- <ranger-admin-username\>
+- <ranger-admin-password\>
+- <ranger-host\>
+```bash 
+curl -u <ranger-admin-username>:<ranger-admin-password> -X POST -H "Accept: application/json" -H "Content-Type: application/json" --data @servicedef.json http://<ranger-host>:6080/service/public/v2/api/servicedef
+```
+6. Login to *Privacera Portal*. The  *datahub*  plugin should be available in  *Access Management -> Resource Policies*  section to create authorization policies for Datahub 
```

Comment:
Log into Apache Ranger...

These steps feel _way_ too coupled to Privacera

Reply:
We don't know the behavior of Apache Ranger Portal and hence it is good to mention Privacera Portal. We will update this line once Privacera or Ranger team resolves the Apache Ranger Portal issue.